public/assets/styles/_sections/Functionlities.less

```diff
@@ -0,0 +1,92 @@
+#Functionlities {
+  padding: 2em 0;
+  #bigPhonePic {
+    display: none;
+  }
+  .functionality {
+    text-align: center;
+    .icon {
+      display: flex;
+      justify-content: center;
+      padding: 1em 0;
+      .iconContainer {
+        background: white;
+        width: 95px;
+        height: 95px;
+        box-shadow: 0 8px 14px rgba(0, 0, 0, 0.16);
+        border-radius: 50px;
+        display: flex;
+        justify-content: center;
+        align-items: center;
+      }
+    }
+    .title {
+      text-transform: uppercase;
+      font-size: 14px;
+      letter-spacing: 1.87px;
+    }
+    .text {
+      color: @light-gray;
+      font-size: 18px;
+      line-height: 30px;
+    }
+  }
+}
+
+
+@media screen and (max-width: 768px) {
+  #slider.swipe {
+    overflow: hidden;
+    visibility: hidden;
+    position: relative;
+    .swipe-wrap {
+      overflow: hidden;
+      position: relative;
+      & > .functionality {
+        float: left;
+        position: relative;
+        overflow: hidden;
+        width: 100%;
+      }
+    }
+  }
+}
+
+@media screen and (min-width: 768px) {
+  #Functionlities {
+    .bigPhonePicContainer {
+      overflow: hidden;
+    }
+    #bigPhonePic {
+      width: 75%;
+      margin: 0 auto;
+      display: block;
+      text-align: center;
+      overflow: hidden;
+      visibility: hidden;
+      position: relative;
+      .swipe-wrap {
+        overflow: hidden;
+        position: relative;
+        & > img {
+          float: left;
+          position: relative;
+          overflow: hidden;
+          width: 100%;
+        }
+      }
+    }
+    .functionality {
+      text-align: left;
+      cursor: pointer;
+      opacity: 0.5;
+      transition: all ease .5s;
+      &.active {
+        opacity: 1;
+      }
+      .phonePic{
+        display: none;
+      }
+    }
+  }
+}
```

Comment:
The responsive layout here seems to be broken. When I resize the page the layout breaks. Trying to make swipe completely responsive can be tricky, so let me know if you have any questions.